webapp/tests/forms/test_session_data.py

```diff
@@ -40,15 +25,15 @@ def test_if_session_data_and_default_data_different_then_update_session_data(sel
         session_data = get_session_data('form_data', default_data=default_data)
         assert expected_data == session_data
 
-    def test_if_session_data_for_the_current_user_is_returned(self, testing_database):
+    def test_if_session_data_for_the_current_user_is_returned(self, testing_current_user):
         form_data = {"brother": "Luigi"}
         other_form_data = {"enemy": "Bowser"}
         expected_data = {**other_form_data}
         override_session_data(form_data)
-        testing_database.setattr(CURRENT_USER, MagicMock(idnr_hashed="9876543210"))
-        override_session_data(other_form_data)
-        session_data = get_session_data('form_data')
-        assert expected_data == session_data
+        with patch(CURRENT_USER, MagicMock(idnr_hashed="9876543210")):
```

Comment:
Why do we need that if we already use the fixture?

Reply:
Because we are changing the current user, so we create another entry in redis different than the previous user